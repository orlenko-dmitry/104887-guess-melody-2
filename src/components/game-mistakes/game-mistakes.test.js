import React from 'react';
import {render} from '@testing-library/react';

import GameMistakes from '../game-mistakes/game-mistakes.jsx';

it(`GameMistakes renders correctly`, () => {
  const {container} = render(<GameMistakes mistakes={0} />);

  expect(container.firstChild).toMatchSnapshot();
});
