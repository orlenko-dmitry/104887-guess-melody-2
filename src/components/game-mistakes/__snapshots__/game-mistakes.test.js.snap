// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameMistakes renders correctly 1`] = `
<div
  class="game__mistakes"
/>
`;
